server/mlabns/handlers/lookup.py

```diff
@@ -23,7 +23,7 @@ def _create_tool_url(fqdn, http_port):
     Returns:
         A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (fqdn, http_port)
+    return str('http://%s:%s' % (fqdn, http_port))
```

Comment:
Put a comment above it saying
  `# Use str() to ensure we return a non-unicode string` 
and I am happy.